components/sidebar/legacy_sidebar_header/dropdown/sidebar_header_dropdown.tsx

```diff
@@ -3,12 +3,14 @@
 
 import React from 'react';
 
+import IconButton from '@mattermost/compass-components/components/icon-button';
+
```

Comment:
I believe this is a left-over remnant that now isn't being used after you merged master?

```suggestion
```